t/27-consoles-s3270.t

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/perl
+
+use Data::Dumper;
+use Test::Most;
+use Mojo::Base -strict, -signatures;
+use Test::Mock::Time;
+use Test::MockModule;
+use Test::MockObject;
+use Test::Output qw(combined_like stderr_like);
+use Test::Warnings qw(:all :report_warnings);
+use POSIX qw(waitpid _exit);
+use FindBin '$Bin';
+use lib "$Bin/../external/os-autoinst-common/lib";
+use OpenQA::Test::TimeLimit '5';
+use consoles::console;
+use backend::ipmi;
+use consoles::VNC;
+use consoles::s3270;
+use cv;
+
+cv::init;
+require tinycv;
+
+$bmwqemu::topdir = "$Bin/..";
+$bmwqemu::vars{ZVM_HOST} = "localhost";
+$bmwqemu::vars{ZVM_GUEST} = "guest";
+$bmwqemu::vars{ZVM_PASSWORD} = "password";
+
+my $ipc_run_mock = Test::MockModule->new('IPC::Run');
+my $vnc_mock = Test::MockModule->new('consoles::VNC');
+my $inet_mock = Test::MockModule->new('IO::Socket::INET');
+my $s = Test::MockObject->new->set_true(qw(sockopt fileno print connected close blocking));
+# $s->mock(read => sub { $_[1] = $s->mocked_read; length $_[1] });
+$vnc_mock->redefine(_read_socket => sub { substr(${$_[1]}, $_[3], $_[2]) = $s->mocked_read; length ${$_[1]} });
+$vnc_mock->redefine(login => 1);
+$inet_mock->redefine(new => $s);
+$ipc_run_mock->redefine(start => sub ($self, $in, $out, $err) {
+        $$out = "success\nconnet(localhost)\nok";
+        return bless({'KIDS' => [{'VAL' => $$in, 'PID' => '0', 'NUM' => 1, 'TYPE' => 'cmd', 'RESULT' => 1, 'OPS' => []}]}, 'IPC::Run');
+});
+$ipc_run_mock->redefine(pump => 1);
+
+ok my $backend_base = backend::baseclass->new(), 'baseclass can be created';
```

Comment:
this test file is about s3270 so we shouldn't need to test creation of baseclass. I suggest you just write
```suggestion
my $backend_base = backend::baseclass->new();
```